src/cls/IPM/Storage/Module.cls

```diff
@@ -85,43 +85,54 @@ Property AfterInstallMessage As %String(MAXLEN = "", XMLPROJECTION = "Element");
 
 Property Deployed As %Boolean(XMLPROJECTION = "Element");
 
-// Relationship UpdateSteps As %IPM.Storage.UpdateStep(XMLPROJECTION = "NONE") [ Cardinality = one, Inverse = manyProp, OnDelete = cascade ];
+Relationship UpdateSteps As %IPM.Storage.UpdateStep(XMLPROJECTION = "NONE") [ Cardinality = many, Inverse = ModuleHome ];
 
 Property UpdatePackage As %String(MAXLEN = 255);
 
-ClassMethod AddUpdateStep(
+/// Add an update step and connect it to this module
+Method AddUpdateStep(
 	className As %String,
-	methodName As %String) As %Status
+	methodName As %String) As %IPM.Storage.UpdateStep
 {
-	return ##class(%IPM.Storage.UpdateStep).%New(className, methodName)
+	set newStep = ##class(%IPM.Storage.UpdateStep).%New(className, methodName)
+	do ..UpdateSteps.Insert(newStep)
+	do ..%Save()
+	return newStep
```

Comment:
Nit: Use formatting settings in the vscode settings file which sets tab size to 4 so not sure why it became 2 for these files